app/lib/tcomb-form/factories/LocalImageFactory.js

```diff
@@ -0,0 +1,33 @@
+import t from 'tcomb-form-native';
+import ImagePicker from 'react-native-image-picker';
+
+import localImageTemplate from '../templates/LocalImageTemplate';
+
+const Component = t.form.Component;
+
+export default class LocalImageFactory extends Component {
```

Comment:
What about something like (before it is extracted as a package):
`app/components/tcomb-form/LocalImage/{Factory,Template}.js`
`app/components/tcomb-form/MaskedInput/Template.js`